src/support_sphere/lib/auth/login/login.dart

```diff
@@ -0,0 +1,147 @@
+                Row(mainAxisAlignment: MainAxisAlignment.center, children: [
+                  const Text('Don\'t have an account?'),
+                  const SizedBox(width: 5.0),
+                  GestureDetector(
+                    onTap: () {
+                      Navigator.of(context).push(
+                        MaterialPageRoute(
+                          builder: (_) => Register(),
+                        ),
+                      );
+                    },
+                    child: Text(
+                      'Sign Up',
+                      style: TextStyle(
+                        fontWeight: FontWeight.bold,
+                        color: Theme.of(context).colorScheme.secondary,
+                      ),
+                    ),
+                  ),
+                ]),
+              ]),
+        ));
+  }
+
+  buildForm(BuildContext context) {
+    return Form(
+        child: Column(
+      children: [
+        const TextFormBuilder(
+          enabled: true,
+          prefix: Ionicons.mail_outline,
+          hintText: "Email",
+          textInputAction: TextInputAction.next,
+        ),
+        const SizedBox(height: 10.0),
+        const PasswordFormBuilder(
+          enabled: true,
+          prefix: Ionicons.lock_closed_outline,
+          suffix: Ionicons.eye_outline,
+          hintText: "Password",
+          obscureText: true,
+          textInputAction: TextInputAction.done,
+        ),
+        Padding(
+          padding: const EdgeInsets.only(right: 10.0),
+          child: Column(
+            children: [
+              Align(
+                  alignment: Alignment.centerRight,
+                  child: InkWell(
+                    onTap: () {
+                      print("Forgot password clicked");
+                    },
+                    child: const Text(
+                      'Forgot Password?',
+                      style: TextStyle(
+                        fontWeight: FontWeight.bold,
+                      ),
+                    ),
+                  ))
+            ],
+          ),
+        ),
+        const SizedBox(height: 10.0),
+        Container(
+          height: 45.0,
+          width: 180.0,
+          child: ElevatedButton(
+            style: ButtonStyle(
+              shape: WidgetStateProperty.all<RoundedRectangleBorder>(
+                RoundedRectangleBorder(
+                  borderRadius: BorderRadius.circular(40.0),
+                ),
+              ),
+              backgroundColor: WidgetStateProperty.all<Color>(
+                Theme.of(context).colorScheme.primary,
+              ),
+            ),
+            // highlightElevation: 4.0,
+            child: const Text(
+              "LOGIN",
+              style: TextStyle(
+                color: Colors.white,
+                fontSize: 12.0,
+                fontWeight: FontWeight.w600,
+              ),
+            ),
+            onPressed: () => {print("Login submitted")},
```

Comment:
Are these print statements just placeholders for now?

Reply:
Yes indeed 😄